modules.d/00systemd/module-setup.sh

```diff
@@ -47,7 +47,6 @@ install() {
         "$systemdutildir"/system-generators/systemd-debug-generator \
         "$systemdutildir"/system-generators/systemd-fstab-generator \
         "$systemdutildir"/system-generators/systemd-gpt-auto-generator \
-        \
```

Comment:
If these lines are not removed, the output of shfmt would be the followings:

`-        \
-        "$systemdsystemunitdir"/debug-shell.service \
+        "\
+$systemdsystemunitdir"/debug-shell.service \`